pkgs/development/web/cypress/default.nix

```diff
@@ -67,7 +67,7 @@ in stdenv.mkDerivation rec {
     printf '{"version":"%b"}' $version > $out/bin/resources/app/package.json
     # Cypress now looks for binary_state.json in bin
     echo '{"verified": true}' > $out/binary_state.json
-    ln -s $out/opt/cypress/Cypress $out/bin/Cypress
+    ln -s $out/opt/cypress/Cypress $out/bin/cypress
```

Comment:
Make a wrapper which warns to tell the user to use the lowercase executable

Can be done with makeWrapper and `--run` https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+%2F--run.%2Becho%2F&type=code